### PR TITLE
python3.3 support dropped

### LINKED ISSUE #99: Drop support for python 3.3
It has reached end-of-life
https://www.python.org/downloads/release/python-337/ on 2017-09-29

### PR DESCRIPTION
**Pull request by grzanka**
Fixes #99

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,8 +10,6 @@ matrix:
     env: TOXENV=coverage
   - python: 3.2
     env: TOXENV=py32 VENVVER="<14.0.0" PIPVER="<8.0.0" STVER="<30.0.0"
```
Comment (hugovk) [inline .travis.yml]: If 3.3 is EOL, 3.2 is even more EOL! Drop that too.

**Comment by hugovk** [comment]
It'd be good to add something like this to make sure pip installs the correct version for the user's Python version.

```python
python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*',
```

File: .travis.yml (hunk 1 above)
Comment (grzanka) [reply .travis.yml]: We still have (unfortunately) some users running Debian 7 which has Python 3.2 installed. I'd delay dropping python3.2 until May 2018 when Debian 7 will be EOL.
I'll do it gladly !

**Comment by grzanka** [comment]
@hugovk python_requires implemented in 7579aea
I am curious if you are also working in same field as me - are you using both pymchelper and mcpartools ? Can we communicate by some other channel ?

File: setup.py [hunk 2]
```diff
@@ -151,7 +151,6 @@ def get_cmdclass():
         # that you indicate whether you support Python 2, Python 3 or both.
         'Programming Language :: Python :: 2.7',
         'Programming Language :: Python :: 3.2',
-        'Programming Language :: Python :: 3.3',
         'Programming Language :: Python :: 3.4',
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
```
Comment (ant6) [inline setup.py]: I wonder, why there is no 3.6 in tox.ini?

Comment (grzanka) [reply setup.py]: fixed in a84d3b6

**Comment by hugovk** [comment]
@grzanka No, I'm not working in this field, I just saw the open issues and am trying to help the Python community move away from old, EOL versions :)

**Comment by grzanka** [comment]
@hugovk Thank you then for spotting this one ! I appreciate your care.

**Pull request merged**